go.mod1

```diff
@@ -15,3 +15,5 @@ require (
 	golang.org/x/net v0.0.0-20181029044818-c44066c5c816
 	golang.org/x/text v0.3.0
 )
+
+replace github.com/pingcap/tidb => github.com/leoppro/tidb v0.0.0-00000000000000-4efb42782bbc7b9589ebf4afb2f82416c7a76fa6
```

Comment:
Why this change ?

Reply:
we can't compile successfully if we didn't change the go.mod because we changed the Node interface.

Reply:
after the pr (pingcap/tidb#8679) merged, we can change it back.